pages/api/send-analytics.ts

```diff
@@ -7,7 +7,7 @@ const { serverRuntimeConfig } = getConfig();
 
 export default async function sendAnalytics(req: NextApiRequest, res: NextApiResponse) {
     const {event, data = {}} = req?.body;
-    const anonymousId = req.cookies['helio_analytics_id'] || moment().unix().toString()
+    const anonymousId = req.cookies['user_id'] || req.cookies['helio_analytics_id'] || moment().unix().toString()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the way `anonymousId` is determined could potentially lead to a situation where the same user gets assigned different IDs in different sessions, if they have a `user_id` cookie in one session but not in another. This could affect the accuracy of analytics data. To avoid this, consider maintaining consistency in the method of assigning `anonymousId`.

```diff
- const anonymousId = req.cookies['user_id'] || req.cookies['helio_analytics_id'] || moment().unix().toString()
+ const anonymousId = req.cookies['helio_analytics_id'] || req.cookies['user_id'] || moment().unix().toString()
```
This way, `helio_analytics_id` will be prioritized over `user_id` when both are present, ensuring that the same ID is used across sessions for the same user.


<!-- This is an auto-generated comment by OSS CodeRabbit -->